modules/nf-neuro/utils/dwishell/main.nf

```diff
@@ -0,0 +1,63 @@
+process UTILS_DWISHELL {
+    tag "${meta.id}"
+    label 'process_single'
+
+    container "scilus/scilpy:2.2.0_cpu"
+
+    input:
+    tuple val(meta), path(dwi), path(bval), path(bvec)
+
+    output:
+    tuple val(meta), path("*__dwi_sh_shells.nii.gz"), path("*__bval_sh_shells"), path("*__bvec_sh_shells"), emit: dwi_shells
```

Comment:
split this in 3 outputs (dwi, bval, bvec). We try to follow the `nf-core` convention of **1 file per output channel** whenever possible since it makes modules more generic and their usage subworkflows and pipelines more readable.